internal/build/pom.xml

```diff
@@ -1054,5 +1057,21 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <!-- Profile to skip time-consuming steps. -->
+            <id>fast</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <basepom.check.skip-all>true</basepom.check.skip-all>
+                <!-- skip _compiling_ the tests -->
+                <maven.test.skip>true</maven.test.skip>
```

Comment:
this skips the test compilation; this is a habit we should not get into.

Reply:
I'll update the PR. There isn't much of it left however 😂

Reply:
you could add another target to the makefile that allows "make install-fast"

Reply:
> you could add another target to the makefile that allows "make install-fast"

good point!
done